src/progress/Progress.ts

```diff
@@ -28,15 +25,15 @@ export const useProgress = createHook<ProgressOptions, ProgressHTMLProps>({
   keys: PROGRESS_KEYS,
 
   useProps(options, htmlProps) {
-    const { isIndeterminate, value, max, min, ariaValueText } = options;
+    const { isIndeterminate, value, max, min } = options;
 
     return {
       role: "progressbar",
       "data-indeterminate": dataAttr(isIndeterminate),
       "aria-valuemax": max,
       "aria-valuemin": min,
       "aria-valuenow": isNull(value) ? undefined : value,
-      "aria-valuetext": `${ariaValueText}`,
+      "aria-valuetext": !value?.toString() ? "intermediate" : value.toString(),
```

Comment:
value is gonna be number right? can we just check with isNaN?

Reply:
Yeah, I checked both does the same thing right !! 🤔